source/excerptexport/models/bounding_geometry.py

```diff
@@ -1,40 +1,67 @@
+from django.conf import settings
 from django.contrib.gis.db import models
 from django.contrib.gis.geos import GEOSGeometry, Polygon
-from django.utils.translation import gettext_lazy as _
+from django.core.files.storage import FileSystemStorage
 
-from django_enumfield import enum
 
+class BoundingGeometry(models.Model):
+    pass
 
-class BoundingGeometryType(enum.Enum):
-    BOUNDINGBOX = 0
-    # POLYGON = 1
 
+class OsmosisPolygonFilterBoundingGeometry(BoundingGeometry):
+    """
+    Bounding geometry based on a 'Osmosis polygon filter file format' file.
+    """
+    private_storage = FileSystemStorage(location=settings.PRIVATE_MEDIA_ROOT)
 
-class BoundingGeometry(models.Model):
-    @staticmethod
-    def create_from_bounding_box_coordinates(north, east, south, west):
-        bounding_geometry = BoundingGeometry()
-        bounding_geometry.type = BoundingGeometryType.BOUNDINGBOX
+    # FIXME: Serialization hardcodes the storage's location into the generated migration as absolute path,
+    #        thus making the project unportable.
+    polygon_file = models.FileField(storage=private_storage)
+
+
+class BBoxBoundingGeometry(BoundingGeometry):
+    south_west = models.PointField()
+    north_east = models.PointField()
+
+    @property
+    def west(self):
+        return self.south_west[0]
+
+    @property
+    def east(self):
+        return self.north_east[0]
+
+    @property
+    def north(self):
+        return self.north_east[1]
+
+    @property
+    def south(self):
+        return self.south_west[1]
+
+    @property
+    def geometry(self):
         polygon = Polygon([
-            GEOSGeometry('POINT(%s %s)' % (west, south)),
-            GEOSGeometry('POINT(%s %s)' % (west, north)),
-            GEOSGeometry('POINT(%s %s)' % (east, north)),
-            GEOSGeometry('POINT(%s %s)' % (east, south)),
-            GEOSGeometry('POINT(%s %s)' % (west, south))
+            self.south_west,
+            GEOSGeometry('POINT(%s %s)' % (self.west, self.north)),
+            self.north_east,
+            GEOSGeometry('POINT(%s %s)' % (self.east, self.south)),
+            self.south_west
         ])
-        bounding_geometry.geometry = GEOSGeometry(polygon)
-        return bounding_geometry
+        return GEOSGeometry(polygon)
 
-    type = enum.EnumField(BoundingGeometryType, default=BoundingGeometryType.BOUNDINGBOX, verbose_name=_('type'))
-    geometry = models.GeometryField(srid=4326, blank=True, null=True, spatial_index=True, verbose_name=_('geometry'))
```

Comment:
I had to drop this translation as the `geometry` field is replaced by a method of the same name. If we need it, we might have to look into Django's notion of virtual (i.e. computed) fields, which seems to be based on python properties.